src/System.Numerics.Tensors/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <AssemblyVersion>0.1.0.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
Remove these two properties: they control whether this lib is part of shared framework.